_docs/geyser/faq.md

```diff
@@ -75,7 +75,16 @@ If there is a prefix on Floodgate players, you must include the prefix in the na
 Example: `/tp ".<bedrock_username>"`
 
 ## How do I add players to the whitelist when using Floodgate?
-There are three ways you can do this. The first way is to use Floodgate's built-in whitelist command, `/fwhitelist add <bedrock_username>`. The second way is to turn off the whitelist using `/whitelist off`, then get the Geyser player to join, then run `/whitelist add ".<bedrock_username>"`, then turn the whitelist back on using `/whitelist on`. (If using this method for a Bedrock account linked to an existing, whitelisted Java account, there is no need to _also_ whitelist the Bedrock account; you may link the accounts then immediately turn the whitelist back on.) The third way is to add the player's UUID as given by Floodgate to the whitelist.json file and then run `/whitelist reload`.
+
+There are three ways you can do this.
+
+1. Use Floodgate’s built-in whitelist command: `/fwhitelist add <bedrock_username>`.
```

Comment:
```suggestion
1. Use Floodgate’s built-in whitelist command: `/fwhitelist add <bedrock_username>`. This will only work if the player has joined any Geyser server already.
```

Reply:
I would write it as "This will only work if the player has been previously in a Geyser server already."